lms/services/organization_usage_report.py

```diff
@@ -103,7 +138,11 @@ def usage_report(
         # Of those groups, get the ones that do have annotations in the time period
         groups_with_annos = [
             group.authority_provided_id
-            for group in self._h_api.get_groups(groups_from_org, since, until)
+            for group in self._h_api.get_groups(
+                groups_from_org,
+                datetime.combine(since, datetime.min.time()),  # Start of since date
```

Comment:
We now use `date` everywhere in LMS but the H api uses `datetime`, make the conversion here.